include/libopencm3/stm32/f7/memorymap.h

```diff
@@ -93,7 +93,7 @@
 #define SPI6_BASE			(PERIPH_BASE_APB2 + 0x5400)
 #define SAI1_BASE			(PERIPH_BASE_APB2 + 0x5800)
 #define SAI2_BASE			(PERIPH_BASE_APB2 + 0x5C00)
-#define LCD_TFT_BASE			(PERIPH_BASE_APB2 + 0x6800)
+#define LTDC_BASE			(PERIPH_BASE_APB2 + 0x6800)
```

Comment:
it's still called LCD-TFT in the ref man, I'm not really keen on changing this, at least not in the same commit.  I'd be more open to just having a second one perhaps?

Reply:
it could also be changed in the f4 memorymap